core/config.py

```diff
@@ -43,7 +43,7 @@ def __setitem__(self, key, value):
 
 # Some values used more than once below
 __project_id = google.auth.default()[1]
-__version = os.getenv("GAE_VERSION")
+__project_version = os.getenv("GAE_VERSION")
```

Comment:
The project itself has no version. Can we call it `app_version`, as in the conf dictionary?

```suggestion
__app_version = os.getenv("GAE_VERSION")
```

Reply:
I've updated the PR. please re-re-review.